WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -526,10 +532,15 @@ public static int deletePostsWithTag(final ReaderTag tag) {
         }
 
         String[] args = {tag.getTagSlug(), Integer.toString(tag.tagType.toInt())};
-        return ReaderDatabase.getWritableDb().delete(
+        int rowsDeleted = ReaderDatabase.getWritableDb().delete(
```

Comment:
This is an instance in the method was directly returning the int value. I changed to keep the value local and will only post the new event is rows have actually been deleted.